app/models/sequenom_qc_plate.rb

```diff
@@ -77,6 +77,15 @@ def compute_and_set_name(input_plate_names)
     end
     true
   end
+
```

Comment:
Ahh, and good counterpoint to the earlier caution of making private stuff public. In this case its the right thing to do, as the print labels methods you removed quite rightly didn't belong here. So good.